data/json/items/gun/223.json

```diff
@@ -45,6 +45,39 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "fs2000",
+    "copy-from": "rifle_semi",
+    "type": "GUN",
+    "name": "FN FS2000",
+    "description": "A sleek bullpup carbine rifle designed by FN Herstal, complete with an integrated sight accessory rail. The forward-firing ejection port and ambidextrous controls make firing comfortable for both left and right-handed shooting.",
+    "weight": "3565 g",
+    "volume": "6688 mL",
+    "price": 125000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "dispersion": 150,
+    "durability": 7,
+    "min_cycle_recoil": 1350,
+    "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ],
+    "magazine_well": 1,
```

Comment:
```suggestion
    "magazine_well": 1,
```
This could be set to: Volume of stanag30-(Volume of gun with magazine as determined by image analysis")- "Volume of gun without magazine as determined by image analysis"). Hopefully magazine_well takes strings.

Reply:
using width at top = width of ar15 magwell

![image](https://user-images.githubusercontent.com/7764202/72094308-c60a1e80-32db-11ea-8c56-2002c5c1a4ab.png)
![image](https://user-images.githubusercontent.com/7764202/72094357-dde1a280-32db-11ea-828d-dbb4d5746c1f.png)

Stanag 30 rounder has a volume of 12709.192 sq mm * .898 in or 22.8092 mm = 289.88 mL.

Using dimensions given in archived manual.
![image](https://user-images.githubusercontent.com/7764202/72094492-2a2ce280-32dc-11ea-986c-69ee61a6c41d.png)
![image](https://user-images.githubusercontent.com/7764202/72094857-f0101080-32dc-11ea-8bd6-f9594fe144da.png)
7 985.88848 ml for gun, 61.4577699 ml for exposed stanag volume, 289.88- exposed stanag volume; fs 2000 has a magwell = "228 ml"